### PR TITLE
Workbench session image promotion

### PR DESCRIPTION
**Pull request by bschwedler**
Build the `workbench-session` image.

### CODE REVIEW

File: docker-bake.hcl [hunk 1]
```diff
@@ -209,6 +225,8 @@ group "default" {
         "package-manager",
         "r-session-complete",
         "workbench",
+        "workbench-session",
+        "workbench-session-init",
```
Comment (zachhannum) [inline docker-bake.hcl:229]:
```suggestion
```

Can we just remove it from here to prevent it from building?

Comment (bschwedler) [reply docker-bake.hcl:229]: #878 Resolves this at the Actions level.

**Review by zachhannum — APPROVED** [review]

**Comment by bschwedler** [comment]
@zachhannum is it OK if the preview/daily builds still run for the session init container? (as they did in the above workflow)

**Comment by zachhannum** [comment]
> @zachhannum is it OK if the preview/daily builds still run for the session init container? (as they did in the above workflow)

Yep, preview/daily builds are still OK! We are using them for QA prior to the release. Thanks!

**Comment by zachhannum** [comment]
> @zachhannum is it OK if the preview/daily builds still run for the session init container? (as they did in the above workflow)

Yep, preview/daily builds are still OK! We are using them for QA prior to the release. Thanks!

**Pull request merged**